pynitrokey/cli/__init__.py

```diff
@@ -58,10 +61,30 @@ def nitropy():
     handler = logging.FileHandler(filename=LOG_FN, delay=True, encoding="utf-8")
     logging.basicConfig(format=LOG_FORMAT, level=logging.DEBUG, handlers=[handler])
 
+    # Timestamp for log
+    logger.info(f"\nTimestamp: {datetime.now()}")
+    # Info Os
+    logger.info(f"\nOS: {platform.uname()}")
+    # Info Python version
+    logger.info(f"Pythonversion: {sys.version}\n")
```

Comment:
Typo: 
> pythonversion

Also possible:


```suggestion
    logger.info(f"Python version: {sys.version}\n")
    logger.info(f"Python version: {platform.python_version()}\n") # alternative
```

Reply:
Ah, I've missed that - the `\n` is not needed here

Reply:
i will change it.